server/master/src/main/java/org/apache/accumulo/master/tableOps/WriteExportFiles.java

```diff
@@ -144,16 +145,15 @@ public static void exportTable(VolumeManager fs, AccumuloServerContext context,
     BufferedOutputStream bufOut = new BufferedOutputStream(zipOut);
     DataOutputStream dataOut = new DataOutputStream(bufOut);
 
-    try {
+    try (OutputStreamWriter osw = new OutputStreamWriter(dataOut, UTF_8)) {
```

Comment:
This got dinged by Findbugs for not closing the stream so I had put the OutputStreamWriter in the try with resources.